core/templates/domain/story/StoryContentsObjectFactory.ts

```diff
@@ -107,6 +107,24 @@ export class StoryContents {
     return nodeTitles;
   }
 
+  getNodeIdsToDescriptionMap(nodeIds: string[]): {} {
+    var nodes = this._nodes;
+    var nodeDescription = {};
```

Comment:
Rename it to nodeIdsToDescriptionMap.

Reply:
@aks681 just to be sure, getNodeIdsToDescriptionMap function name is similar to other function names in the code above and below. So should I change it?

Reply:
I have removed it as requested in different comment.